packages/api/cms-api/src/dam/files/files.service.ts

```diff
@@ -439,14 +462,17 @@ export class FilesService {
         const fileInput: CreateFileInput & { copyOf: FileInterface } = {
             ...Utils.copy(fileProps),
             image: fileImageInput,
-            folderId: inboxFolder.id,
+            folderId: targetFolder?.id,
             copyOf: file,
-            scope: inboxFolder.scope,
+            scope: targetScope,
         };
 
         return this.create(fileInput);
     }
 
+    /**
+     *  @deprecated Loop over FilesService.createCopyOfFile() instead
+     */
     async copyFilesToScope({ user, fileIds, inboxFolderId }: { user: CurrentUserInterface; fileIds: string[]; inboxFolderId: string }) {
```

Comment:
This method expects a user object and performs permission checks but we agreed that we don't want that in a service. 

Apart from that, the method isn't really useful as it only really loops over `createCopyOfFile()`